### PR TITLE
updated usage guidelines

### PR DESCRIPTION
**Pull request by harsha-simhadri**

### CODE REVIEW

File: workflows/in_memory_index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+To generate index, use the `tests/build_memory_index` program. 
+--------------------------------------------------------------
+
+```
+./tests/build_memory_index  data_type<int8/uint8/float>  dist_fn<l2/mips>   data_file.bin  output_index_file  R(graph degree)   L(build complexity)  alpha(graph diameter control)   T(num_threads)
+```
+
+The arguments are as follows:
+
+(i) **data_type**: The type of dataset you wish to build an index. float(32 bit), signed int8 and unsigned uint8 are supported.
```
Comment (daxpryce) [inline workflows/in_memory_index.md:13]: If you use numbers like:

```
1. **data_type**: ...
2. **dist_fn**: ...
```
You'll get formatting like:

1. **data_type**: The type of dataset you wish to build an index. float(32 bit), signed int8 and unsigned uint8 are supported.
2. **dist_fn**: There are two distance functions supported: minimum Euclidean distance (l2)) and maximum inner product (mips).

I mean, if you want to get all fancified :)

File: workflows/SSD_index.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+To generate an SSD-friendly index, use the `tests/build_disk_index` program. 
+----------------------------------------------------------------------------
+
+```
+./tests/build_disk_index  data_type<float/int8/uint8>   dist_fn<l2/mips>   data_file.bin   index_prefix_path   R(graph degree)   L(build complexity)   B(search memory allocation in GB)   M(build memory allocation in GB)   T(#threads)   PQ_disk_bytes" 
+```
+
+The arguments are as follows:
+
+(i) **data_type**: The type of dataset you wish to build an index. float(32 bit), signed int8 and unsigned uint8 are supported.
```
Comment (daxpryce) [inline workflows/SSD_index.md:13]: In the memory index I mentioned the markdown for lists - just wanted to reference it here too https://github.com/microsoft/DiskANN/pull/48/files#r824174894

File: README.md [hunk 3]
```diff
@@ -83,120 +83,8 @@ Check msbuild docs for additional options including choosing between debug and r
...
-(iv) query_bin: search on these queries, same format as data file (ii) above. The query file must be the same type as specified in (i).
-
-(v) Truthset file. Must be in the following format: n, the number of queries (4 bytes) followed by d, the number of ground truth elements per query (4 bytes), followed by n*d entries per query representing the d closest IDs per query in integer format,  followed by n*d entries representing the corresponding distances (float). Total file size is 8 + 4*n*d + 4*n*d. The groundtruth file, if not available, can be calculated using our program, tests/utils/compute_groundtruth.
-
-(vi) K: search for recall@k, meaning accuracy of retrieving top-k nearest neighbors.
-
-(vii) result output prefix: will search and store the computed results in the files with specified prefix in bin format.
-
-(viii, ix, ...) various search_list sizes to perform search with. Larger will result in slower latencies, but higher accuracies. Must be atleast the recall@ value in (vi).
+- [link](workflows/SSD_index.md) Commandline interface for building and search SSD based indices
```
Comment (daxpryce) [inline README.md:88]: Consider:

```
- [Commandline interface for building and search SSD based indices](workflows/SSD_index.md)  
- [Commandline interface for building and search in memory indices](workflows/in_memory_index.md) 
```

**Review by daxpryce — APPROVED** [review]
My suggestions are inline, none of them are blockers though

File: README.md (hunk 3 above)
Comment (harsha-simhadri) [reply README.md]: changed

**Pull request merged**